### PR TITLE
Closes #99 optha ae aelat@devel

### LINKED ISSUE #99: Add AELAT to AE SDTM for admiralophtha
admiralophtha have created AE.AELAT variable and want it added to `admiral.test`
See code https://github.com/pharmaverse/admiralophtha/blob/devel/dev/ae.R

@bms63 I was thinking to just add this to `admiral_ae` however ophtha team think AELAT is Ophtha specific and maybe data set should be `admiralophtha_ae`. What do you think?

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! 

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. 

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by millerg23** [comment]
@bms63 @rossfarrugia I followed instructions in README file, but I can't generate the `.man` files, it ERRORs each time I run `devtools::document()`. The instructions are out of date, as the programs are in `dev` folder not `inst/dat_scripts`, so maybe there have been other changes?? Any guidance appreciated, thanks.

**Comment by bms63** [comment]
I got `devtools::document()` to work again, but checks still failing...

**Comment by millerg23** [comment]
@bms63 ah "raw_ae" is commented out, thats how `devtools::document()` is working!

**Comment by bms63** [comment]
lol that was a mistake - still playing around

**Comment by millerg23** [comment]
ha, I thought you had worked some magic. In the `data.R` file for `admiral_ae` header should have `#' @source \url{https://github.com/pharmaverse/admiral.test/blob/main/data/admiral_ae.rda}` . I think thats where the checks were failing. But I can't get `devtools::document()` to work. I get:
```
> devtools::document()
Updating admiral.test documentation
ℹ Loading admiral.test
Error: 'raw_ae' is not an exported object from 'namespace:admiral.test'
```

Thanks for looking into this.

**Comment by millerg23** [comment]
@bms63 I re-created the "raw_ae" dataset in the console, and `devtools::document()` worked!
Just waiting on the checks running.
Thanks very much for your help.

**Comment by millerg23** [comment]
@bms63 should I add this to NEWS.md ?

**Comment by bms63** [comment]
Yes please put in something for the news - also nice to link admiralophtha packagedown site in the news :)

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# admiral.test 0.5.0
+ - Updated AE to add variable AELAT for ophthalmology package (#99)
+
 # admiral.test 0.3.0
  - Updated labels for pp and pc datasets (#77)
  - Implemented admiral CI/CD workflows
```
Comment (bms63) [inline NEWS.md:6]:
```suggestion
# admiral.test 0.4.0
* `get_smq_select()` and `get_sdq_select()` have been deprecated in favor of `get_terms()` to accommodate changes in {admiral} (#94)
* Updated labels for PP and PC datasets (#77)
* Implemented admiral CI/CD workflows

# admiral.test 0.3.0
- See Compare of Release between 0.3.0 and 0.20 
```

Comment (bms63) [reply NEWS.md:6]: Hey Gordon.  Just updating news in your PR.  Hope that is okay!

**Review by bms63 — APPROVED** [review]

**Comment by millerg23** [comment]
I saw your message on SLACK, shall we hold off merging to `devel` or we can run downstream checks from `devel`.
There are other datasets to update/add for Ophthalmology.

**Comment by bms63** [comment]
> I saw your message on SLACK, shall we hold off merging to `devel` or we can run downstream checks from `devel`. There are other datasets to update/add for Ophthalmology.

I think this is okay to merge.  It will get tested on our functions and templates when we use @devel in our PRs in those packages anyways.  So we should get info back very quickly.

**Pull request merged**